binr/rafind2/rafind2.c

```diff
@@ -203,11 +212,53 @@ static int rafind_open(char *file) {
 	return 0;
 }
 
+static int rafind_dir_filter(const struct dirent *dp)
+{
+	return (dp->d_name[0] != '.');
+}
+
+static int rafind_open_dir(char *dir) {
+	struct dirent **entries;
+	char *fullpath;
+	int entries_n, i;
+
+	entries_n = scandir(dir, &entries, rafind_dir_filter, alphasort);
+	if (entries_n < 0) {
+		return 1;
+	}
+
+	for (i = 0; i < entries_n; i++) {
+		/* Size of root + found_path + '/' + \0 */
+		fullpath = malloc((strlen(dir) + strlen(entries[i]->d_name + 2) * sizeof(char)));
```

Comment:
use r_str_newf instead of overenegineering with strlens. in fact this is done already in libr/util/file